lib/active_model/serializer/adapter.rb

```diff
@@ -3,40 +3,37 @@ class Serializer
     # @deprecated Use ActiveModelSerializers::Adapter instead
     module Adapter
       class << self
+        extend ActiveModelSerializers::Deprecate
+
         def create(resource, options = {})
```

Comment:
y'know, I tried it with ruby's delegate and forwardable, but not with rails' delegate.  Feel free to try. I think you can push to my branch.